components/Posts.php

```diff
@@ -1,56 +1,74 @@
-<?php namespace RainLab\Blog\Components;
+<?php
+
+namespace RainLab\Blog\Components;
```

Comment:
leave the original namespace formatting please

Reply:
Ok, no problem, will do.